main.go

```diff
@@ -141,14 +149,7 @@ func main() {
 				return err
 			}
 
-			uiPort := uint(rpcPort)
-			// If we're using nats we can't re-use the RPC port for the UI, so we choose another
```

Comment:
definitely good to get rid of port juggling logic as a function of other config options.